tasks/shared_libraries/recovery/README.md

```diff
@@ -60,22 +60,24 @@ the repository.
 
 To use the **shared_recovery** library in your lambda code base perform the following.
 
-1. Install the necessary python libraries shown below.
+1. Keep in mind that the shared library is dependent upon the AWS environment variable "AWS_REGION" and will throw an error if this variable is missing.
```

Comment:
I would add some verbiage that says this is an AWS reserved runtime variable and maybe a link to the documentation. https://docs.aws.amazon.com/lambda/latest/dg/configuration-envvars.html

Reply:
added